purchase_request/models/purchase_request.py

```diff
@@ -9,7 +9,8 @@
     ('draft', 'Draft'),
     ('to_approve', 'To be approved'),
     ('approved', 'Approved'),
-    ('rejected', 'Rejected')
+    ('rejected', 'Rejected'),
+    ('done', 'Done')
```

Comment:
`state = fields.Selection(selection_add=[('done', 'Done')])` is a better way to do it, so in case another module also adds states it won't break.